tests/e2e/tests/devfiles/PHPSimple.spec.ts

```diff
@@ -0,0 +1,75 @@
+/*********************************************************************
+ * Copyright (c) 2020 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+import 'reflect-metadata';
+import * as projectAndFileTests from '../../testsLibrary/ProjectAndFileTests';
+import * as workspaceHandling from '../../testsLibrary/WorksapceHandlingTests';
+import * as commonLsTests from '../../testsLibrary/LsTests';
+import * as codeExecutionTests from '../../testsLibrary/CodeExecutionTests';
+import { e2eContainer } from '../../inversify.config';
+import { WorkspaceNameHandler, Editor, CLASSES } from '../..';
+import { Logger } from '../../utils/Logger';
+
+const editor: Editor = e2eContainer.get(CLASSES.Editor);
+
+const workspaceSampleName: string = 'php-web-simple';
+const fileFolderPath: string = `${workspaceSampleName}`;
+const tabTitle: string = 'index.php';
+// const codeNavigationClassName: string = 'RouterImpl.class';
+const depTaskName: string = 'Configure Apache Web Server DocumentRoot';
+const buildTaskName: string = 'Start Apache Web Server';
+const buildTaskLinkExpectedText: string = 'A process is now listening on port 8080.';
+const stack: string = 'PHP Simple';
+
+suite(`${stack} test`, async () => {
+    suite (`Create ${stack} workspace`, async () => {
+        workspaceHandling.createAndOpenWorkspace(stack);
+        projectAndFileTests.waitWorkspaceReadinessNoSubfolder(workspaceSampleName);
+    });
+
+    suite('Language server validation', async () => {
+        projectAndFileTests.openFile(fileFolderPath, tabTitle);
+        prepareEditorForLSTests();
+        commonLsTests.errorHighlighting(tabTitle, `error_text;`, 14);
+        commonLsTests.suggestionInvoking(tabTitle, 14, 5, 'phpinfo');
+        commonLsTests.autocomplete(tabTitle, 14, 5, 'phpinfo');
```

Comment:
I don't see the value in running those tests that way - they are testing the same... The purpose was to test the autocomplete of variable name and suggestion of method.

Reply:
Oh, I see now. Will try to rewrite the logic.

Reply:
fixed.